hugo/content/info/contrib.md

```diff
@@ -27,23 +44,25 @@ If you have sub-volumes (e.g., long papers, short papers, demonstrations, tutori
 For example, "Proceedings of the 2019 Meeting of the Conference on Empirical Methods in Natural Language Processing (EMNLP): Tutorial Abstracts".
 You should also seek consistency with the names of your volumes from prior years.
 
+If your conference has satellite workshops, please make certain the workshops will actually be run.
+
 With this information in hand, we will assign you a [list of Anthology identifiers](https://docs.google.com/spreadsheets/d/166W-eIJX2rzCACbjpQYOaruJda7bTZrY7MBw_oa7B2E/edit?usp=sharing) for use with ACLPUB.
 
-Ingestion is a multi-step process:
+### Prepare your data
 
-1. **You** arrange your data (PDFs, BibTeX files, and metadata) using ACLPUB by following the instructions below.
+Prepare your data (PDFs, BibTeX files, and metadata) using ACLPUB by following the instructions below.
    You will run a script to lay it out into a single directory, and then send a link to your tarball.
    **This tarball must be delivered to the Anthology director two weeks prior to your desired publication date**, which was negotiated when you first contacted us.
-2. **We** ingest that data by running additional scripts that convert it into our authoritative format and commit it to [our Github repository](https://github.com/acl-org/acl-anthology/).
-3. This becomes a pull request on [our pull requests page](https://github.com/acl-org/acl-anthology/pulls).
-4. Once approved and merged into the `master` branch, the site will be automatically rebuilt (which happens twice a day) and made live.
 
-You are responsible only for the Step 1: producting a minimally compliant XML file along with the associated PDFs and optionally other data (such as software).
 This is handled by the [ACLPUB](https://github.com/acl-org/ACLPUB/) package, whose output is a minimally compliant XML file that can be ingested into [the Anthology's authoritative XML format](#authoritative-xml-format).
 **Instructions for this can be found [in the ACLPUB package](https://github.com/acl-org/ACLPUB/blob/master/anthologize/README.md)**.
 (Additionally, if you are using EasyChair, you will also want to use [our easychair scripts](https://github.com/acl-org/easy2acl), but please start with the ACLPUB documentation).
 
-The remaining steps are handled by Anthology staff and use [Anthology tools](https://github.com/acl-org/acl-anthology/tree/master/bin/).
+The remaining steps are handled by Anthology staff and use [Anthology tools](https://github.com/acl-org/acl-anthology/tree/master/bin/):
+
+- We ingest that data by running additional scripts that convert it into our authoritative format and commit it to [our Github repository](https://github.com/acl-org/acl-anthology/).
+- This becomes a pull request on [our pull requests page](https://github.com/acl-org/acl-anthology/pulls).
+- Once approved and merged into the `master` branch, the site will be automatically rebuilt (which happens twice a day) and made live.
 
 #### Notes about Softconf's START
```

Comment:
I wondered if this entire section could be deleted.

Reply:
Agreed.